api/tests/tests.py

```diff
@@ -404,6 +404,70 @@ def test_get_sessions_by_instructor(self):
         self.assertIn(self.quiz.title, str(response.content))
 
 
+class AddQuizSessionLogTest(BaseTest):
+    def setUp(self):
+        super().setUp()
+        self.valid_payload = {
+            "quiz_session_code": self.new_quiz_session.code,
+            "quiz_session_student_id": self.new_quiz_session_student.id,
+            "question_multiple_choice_id": self.new_question.id,
+            "action": "connected",
+        }
+
+        self.invalid_code_payload = {
+            "quiz_session_code": "INVALID CODE",
+            "quiz_session_student_id": self.new_quiz_session_student.id,
+            "action": "connected",
+        }
+
+        self.invalid_student_payload = {
+            "quiz_session_code": self.new_quiz_session.code,
+            "quiz_session_student_id": 999,
+            "action": "connected",
+        }
+
+        self.invalid_question_payload = {
+            "quiz_session_code": self.new_quiz_session.code,
+            "quiz_session_student_id": self.new_quiz_session_student.id,
+            "question_multiple_choice_id": 999,
+            "action": "connected",
+        }
+
+        self.invalid_action_payload = {
+            "quiz_session_code": self.new_quiz_session.code,
+            "quiz_session_student_id": self.new_quiz_session_student.id,
+            "action": "Invalid Option",
+        }
+
+        self.url = reverse("add-quiz-session-log")
+
+    def test_create_log_entry_success(self):
+        response = self.client.post(self.url, self.valid_payload, format="json")
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(response.data["message"], "Log entry successfully created.")
+        self.assertEqual(QuizSessionLog.objects.count(), 1)
```

Comment:
Once we update the endpoint so that it returns the fields/attributes, we can verify that everything has been stored successfully here.